primefaces/src/main/java/org/primefaces/util/ExcelStylesManager.java

```diff
@@ -251,6 +338,51 @@ private CellStyle createCurrencyStyle() {
         return style;
     }
 
+    private CellStyle createDateStyle() {
+        CellStyle style = createDefaultCellStyle();
+        short dataFormat;
+        if (options.getExcelDateFormat() != null) {
+            String format = options.getExcelDateFormat();
+            dataFormat = wb.getCreationHelper().createDataFormat().getFormat(format);
+        }
+        else {
+            dataFormat = 14;
```

Comment:
Where did this 14 come from?

Reply:
It's a excel built-int format, documented in `org.apache.poi.ss.usermodel.BuiltinFormats`